.flake8

```diff
@@ -0,0 +1,36 @@
+[flake8]
+exclude =
+max-line-length = 88
+inline-quotes = "
+ignore =
```

Comment:
Any reason why we are ignoring these rules? Some context here would help.

Reply:
Copied verbatim from (Ray)[https://github.com/ray-project/ray/blob/master/.flake8]. Should have made that clear